cmd/jaeger/internal/extension/jaegerquery/config.go

```diff
@@ -18,12 +18,20 @@ var _ component.ConfigValidator = (*Config)(nil)
 // Config represents the configuration for jaeger-query,
 type Config struct {
 	queryApp.QueryOptionsBase `mapstructure:",squash"`
```

Comment:
@yurishkuro Do we want/need to rework [this configuration](https://github.com/jaegertracing/jaeger/blob/main/cmd/query/app/flags.go#L60-L78)? A few things I was hoping to get some guidance on: 
- Not all fields have mapstructure tags - are there any here that we want to add tags to? 
- For the fields that don't have mapstructure tags, should we move them to `QueryOptions`? 
- We have `Tenancy` twice